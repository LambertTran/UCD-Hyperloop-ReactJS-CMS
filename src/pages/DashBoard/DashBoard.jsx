import React from 'react';

export default function DashBoard() {
  return (
    <div className="dashboard">
      <div>Navbar</div>
      <h1>WellCome</h1>
      <div>Sub team cards</div>
    </div>
  )
}