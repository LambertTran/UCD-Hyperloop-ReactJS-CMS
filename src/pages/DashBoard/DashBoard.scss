.dashboard{
  margin: 0 auto;
  padding-top: 100px;
  max-width: 1300px;
  @include flexCol;
  justify-content: center;
  align-items: center;
  
  h1{
    text-align: center;
  }
}

.dashboard-teams{
  @include flexCol;
  flex-direction: row;
  flex-wrap: wrap;
  padding: 40px;
  justify-content: center;
}
