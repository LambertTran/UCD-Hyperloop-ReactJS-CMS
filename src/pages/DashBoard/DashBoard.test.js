// packages
import React from 'react';
import { shallow } from 'enzyme';
import setupTests from '../../shares/setupTests';

// components
import DashBoard from './DashBoard';

describe('DashBoard', () => {
  const dashboard = shallow( <DashBoard /> );

  it('renders correctly', () => {
    expect(dashboard).toMatchSnapshot();
  })

  
})