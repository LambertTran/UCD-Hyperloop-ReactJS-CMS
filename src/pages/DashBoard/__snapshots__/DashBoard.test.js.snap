// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashBoard renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DashBoard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div>
          Navbar
</div>,
        <h1>
          WellCome
</h1>,
        <div>
          Sub team cards
</div>,
      ],
      "className": "dashboard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Navbar",
        },
        "ref": null,
        "rendered": "Navbar",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "WellCome",
        },
        "ref": null,
        "rendered": "WellCome",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sub team cards",
        },
        "ref": null,
        "rendered": "Sub team cards",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            Navbar
</div>,
          <h1>
            WellCome
</h1>,
          <div>
            Sub team cards
</div>,
        ],
        "className": "dashboard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Navbar",
          },
          "ref": null,
          "rendered": "Navbar",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "WellCome",
          },
          "ref": null,
          "rendered": "WellCome",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sub team cards",
          },
          "ref": null,
          "rendered": "Sub team cards",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
